### PR TITLE
Test/ui test cases (Implemented #22) Please, review and merge this first

### PR DESCRIPTION
**Pull request by bkrmadtya**
### UI test cases, Implemented #22
UI test cases with Espresso for the following activities:

- Login Activity
- Register Activity
- Menu Activity
- Add Activity
- Delete Activity
- Update Activity

** Test for Delete activity fails for the reason Anna mentioned in #34**

Note:
I will add the rest of the test cases soon.

### CODE REVIEW

File: app/src/androidTest/java/com/example/myapplication/MenuActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Rule
+    public ActivityTestRule<MenuActivity> mActivityRule = new ActivityTestRule<>(MenuActivity.class);
+
+    @Test
+    public void buttonsAreClickable(){
+        Espresso.onView(withId(R.id.btn_add))
+                .check(matches(isClickable()));
+
+        Espresso.onView(withId(R.id.btn_update))
```
Comment (TESLAA) [inline app/src/androidTest/java/com/example/myapplication/MenuActivityTest.java]: You should separate these functionalities, one method for every single espresso check/assertion
Those methods should get named like test[IsUpdateButton]Clickable -> for example

File: app/src/androidTest/java/com/example/myapplication/UpdateActivityTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+        // And the input field for contact name is disabled
+        contact_name_field_isDisabled();
+
+        // input field for contact number is displayed
+        number_field_isDisplayed();
+    }
+
+    @Test
+    public void contact_name_field_isEnabled(){
```
Comment (TESLAA) [inline app/src/androidTest/java/com/example/myapplication/UpdateActivityTest.java]: change the naming as mentioned in previous comment

File: app/src/main/java/com/example/myapplication/AddActivity.java [hunk 3]
```diff
@@ -22,7 +22,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.add_phone_user);
         findViews();
-
+
```
Comment (TESLAA) [inline app/src/main/java/com/example/myapplication/AddActivity.java]: reformat the code here,
so many useless new line and indentations

**Comment by bkrmadtya** [comment]
Refactored and added new tests.

**Comment by TESLAA** [comment]
> Refactored and added new tests.

Please resolve the discussion reviews which we open when we write comment on the code, then next time the one comes to review it's easier and cleaner to find our are they fixed or not yet.
Kiitos

**Pull request merged**